### DISCUSSION TITLE
这个自动构建release note 是怎么加入不属于renovate发布的日志的

### DISCUSSION THREAD

**Discussion by mobeicanyue** [zdz/ServerStatus-Rust#229] [Q&A] [answered]
如图 蓝色的是renovate 发布的 
红色的是手动加进去的还是自动构建的呢？
![IMG_20230705_185449](https://github.com/zdz/ServerStatus-Rust/assets/81098819/2faa27c2-0f20-48e5-ae8b-b80d3f077e26)

**Comment by zdz** [comment, marked as answer]
手动编辑

**Reply by mobeicanyue** [reply]
审核一下pr吧
